mitodl/hugo-course-publisher
# Video pages

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/Kg66zdss/100-create-markdown-files-for-video-pages

#### What's this PR do?
This PR updates markdown with latest changes from source data and add video pages.

#### How should this be manually tested?
Run the site, then try and visit one of the video pages manually by navigating to the section it is supposed to be in.  Copy the title of a video section and append it to the end of your URL.  You should see the title of the video, but no player yet since the template has not been implemented.

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/12089658/77944406-ee773880-728c-11ea-9a86-ead01846c595.png)
